installation/docker.adoc

```diff
@@ -10,4 +10,6 @@
 
 toc::[]
 
-== Overview
\ No newline at end of file
+== Overview
+
+Starting in a later beta release, you will be able to install OpenShift Enterprise 3.0 Beta using Docker images. Images will be made available for OpenShift platform components such as masters, etcd, nodes, and routers.
```

Comment:
@nhr Thanks, I had everything more conditionalized at one point, but wasn't sure how much we needed to do so during the beta period, so I rolled it back for this first cut. I'll go through and add em back in.